### PR TITLE
Adding new container for TPMCalculator

### PR DESCRIPTION
**Pull request by r78v10a07**

### CODE REVIEW

File: TPMCalculator/0.0.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ENV DEPFOLDER=/home/biodocker/
+ENV DST=/tmp
+ENV BAMTOOLS_DIR=/home/biodocker/usr
+ENV CPPFLAGS="-I $BAMTOOLS_DIR/local/include/bamtools"
+ENV LDFLAGS="-L $BAMTOOLS_DIR/local/lib -Wl,-rpath,$BAMTOOLS_DIR/local/lib"
+
+RUN cd $DST && \
+	git clone $URL && \
+	cd $FOLDER && \
+	git clone https://github.com/pezmaster31/bamtools && \
```
Comment (bgruening) [inline TPMCalculator/0.0.1/Dockerfile]: This is cloning bamtools and not TPMCalculator, is this intended?

**Comment by r78v10a07** [comment]
No, this software use bamtools library inside, that’s way I need to compile
bamtools before compiling the TPMCalculator



--

Roberto Vera Alvarez, PhD


Profiles:

LinkedIn <http://www.linkedin.com/in/robertoveraalvarez>, ResearchGate
<http://www.researchgate.net/profile/Roberto_Vera_Alvarez>, Google Citations
<http://scholar.google.it/citations?user=X9OKjuEAAAAJ>, Twitter
<http://twitter.com/r78v10a07>

Skype user: r78v10a07

**Comment by bgruening** [comment]
In this case it might be better to base the container on: https://hub.docker.com/r/biocontainers/bamtools/
ping @ypriverol

**Comment by r78v10a07** [comment]
I try what you’re recommending but my program does not compile. I don’t
know yet if it is due to the fact that the Bamtools version in the
container is the 2.4.0 and not the current one 2.4.1

Let’s put on hold the pull request until I figure out the source of the
error.

Thanks,
Roberto



--

Roberto Vera Alvarez, PhD


Profiles:

LinkedIn <http://www.linkedin.com/in/robertoveraalvarez>, ResearchGate
<http://www.researchgate.net/profile/Roberto_Vera_Alvarez>, Google Citations
<http://scholar.google.it/citations?user=X9OKjuEAAAAJ>, Twitter
<http://twitter.com/r78v10a07>

Skype user: r78v10a07

**Comment by r78v10a07** [comment]
Hi,

I’ve created a new container for the bamtools version 2.4.1 and now my
software compiles well using the bamtools as a base image.

I will commit all changes to the sandbox.

Best,
Roberto



--

Roberto Vera Alvarez, PhD


Profiles:

LinkedIn <http://www.linkedin.com/in/robertoveraalvarez>, ResearchGate
<http://www.researchgate.net/profile/Roberto_Vera_Alvarez>, Google Citations
<http://scholar.google.it/citations?user=X9OKjuEAAAAJ>, Twitter
<http://twitter.com/r78v10a07>

Skype user: r78v10a07

**Comment by r78v10a07** [comment]
I'm adding a new container for Bamtools version 2.4.1 and two more containers which use as base image the bamtools image

**Pull request merged**